dotnet/src/SemanticKernel.Abstractions/Search/TextSearchOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel.Search;
+
+/// <summary>
+/// Options which can be applied when using <see cref="ITextSearch"/>.
+/// </summary>
+[Experimental("SKEXP0001")]
+public sealed class TextSearchOptions
+{
+    /// <summary>
+    /// The name of the search index.
+    /// </summary>
+    public string? Index { get; set; }
```

Comment:
Index seems very vector search specific.  Do we just ignore this for search engines?  Does it make sense to include it at all at this level of the stack then?

Reply:
Will leave it out as it would be configured when we construct the implementation.